src/main/resources/alma/fix/otherFields.fix

```diff
@@ -159,3 +159,9 @@ do list(path: "542??", "var": "$i")
   copy_field("$i.u", "license[].$last.label")
   replace_all("license[].$last.label", "^(http://|https://)(.*)$","$2")
 end
+
+set_array("langNote[]")
+do list(path: "546??", "var": "$i")
+  copy_field("$i.a", "langNote[].$append")
+  copy_field("$i.b", "langNote[].$last")
```

Comment:
So this is _either_ `$$a` or `$$b`? I'm not sure if this is the way to go, but I don't have a counterexample either.

According to the [MARC standard](https://www.loc.gov/marc/bibliographic/bd546.html), there _can_ be both at the same time. So maybe something like this?

```suggestion
  set_array("$i.@langNote")
  copy_field("$i.a", "$i.@langNote.$append")
  copy_field("$i.b", "$i.@langNote.$append")
  replace_all("$i.@langNote.*", "[.;]$", "")
  join_field("$i.@langNote", "; ")
  move_field("$i.@langNote", "langNote[].$append")
```